### ISSUE TITLE
Naming of Notes

### ISSUE THREAD

**Issue by GeoHigh125** [geopaparazzi/geopaparazzi#555]
I have created several complex forms and there is a difference how the notes are named in the notelist.
Sometimes it is the general name following the sectionname, sometimes it takes the first mandatory field, but I cannot find out why.
I have added the first part of two complex forms both seem to be the same but the second one does not deliver a specific name based on the first mandatory field. 
If you have along list of notes it is fine to have the possibility to name them 
![json_id](https://user-images.githubusercontent.com/9881088/43706939-00bebd8e-9967-11e8-87c1-9f17b6db63f5.PNG)
![json_ks](https://user-images.githubusercontent.com/9881088/43706963-120eabf8-9967-11e8-9d73-aca86f211155.PNG)

**Comment by moovida** [comment]
Hi @GeoHigh125 , can you please try to useth ekey and label approach? For complex names or names that can be repeated, that is necessary.
Have a look at section: 13.1.3. Text with key and label in the chapter http://geopaparazzi.github.io/geopaparazzi/#_using_form_based_notes

Let me know if this solves your problem.

**Comment by GeoHigh125** [comment]
Hi @moovida, 
Thanks for the quick reply. I looked for it and i found out, the label key issue is one thing to take care of. I have changed that I use now for the keys something short  (InboetNr) and/or with underscore and for the label then the long description. I will use that for the whole database and keep it in mind for the future form development.
But that did not solve the whole issue. I have added now the "islabel": "true".
The combination now of proper key - label and islabel solved mu problem.
I have added that version.
![json_ks_new](https://user-images.githubusercontent.com/9881088/43712318-070778ca-9976-11e8-943f-2c5ad3031961.PNG)
